charts/sgx-device-plugin/templates/sgx.yaml

```diff
@@ -10,7 +10,6 @@ metadata:
   annotations: {{ toYaml .Values.annotations | nindent 4 }}
 spec:
   image: "{{ .Values.image.hub }}/intel-sgx-plugin:{{ .Values.image.tag | default .Chart.AppVersion }}"
-  initImage: "{{ .Values.initImage.hub }}/intel-sgx-initcontainer:{{ .Values.initImage.tag | default .Chart.AppVersion }}"
```

Comment:
I'd prefer we drop the initContainer starting from `v0.27.0` and we have not released this option yet.